src/sentry/lang/native/minidump.py

```diff
@@ -60,6 +60,20 @@ def process_minidump(minidump, cfi=None):
         return ProcessState.from_minidump_buffer(minidump, cfi)
 
 
+def write_minidump_dummy_data(data):
```

Comment:
Please use the implementation from `wip/symbolicator`:

```py
def add_minidump_payload(data, name=None):
    # Minidump events must be native platform.
    data['platform'] = 'native'

    # Assume that this minidump is the result of a crash and assign the fatal
    # level. This can be overridden in processing, in case the minidump was
    # requested manually.
    setdefault_path(data, 'level', value='fatal')

    # Create a placeholder exception. This signals normalization that this is an
    # error event and also serves as a placeholder if processing of the minidump
    # fails.
    exception = {
        'type': 'Minidump',
        'mechanism': {
            'type': 'minidump',
            'handled': False,
            'synthetic': True,
        }
    }
    data['exception'] = {'values': [exception]}

```

Reply:
I did this now, except for setting `level` which turns out not to be necessary. So users can now set level and we honor it.

Reply:
If you don't set level here, we might not set a level at all if minidump processing fails. It might be preferable for those events to have a default level of `fatal` in that case, as minidumps are almost always process crashes.

Reply:
The function name `dummy` is not very nice, but we can also change that in the subsequent refactor.

Reply:
Both done